cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -902,10 +902,11 @@ body {
                     }
                 }
 
+                //Adjusts position by 2px to accommodate longer titles
                 &-icon-closed .cf-icon-svg,
                 &-icon-open .cf-icon-svg {
                     position: absolute;
-                    right: unit( 24px / @base-font-size-px, em );
+                    right: unit( ( 24px - 3.2 ) / @base-font-size-px, em );
```

Comment:
Should the 3.2 include units? Also, why does the code comment say 2px and this is a fractional value?

Reply:
Arithmetic in less will default to the first units used, I could include them for consistency though here and below. The reason this is 3.2 is kinda convoluted. The computed font-size is 10px, meaning `em` keys off 10px and therefore 0.1 is equivalent to 1px. 3.2/16 (base-font-size-px) = 0.2em or 2px.

Reply:
Would using `rem` make more sense perhaps?